modules/ai/assets/js/editor/pages/form-layout/context/remote-config.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { getRemoteConfig } from '../../../api';
+
+const RemoteConfigContext = React.createContext( {} );
+
+export const useRemoteConfig = () => React.useContext( RemoteConfigContext );
+
+export const RemoteConfigProvider = ( props ) => {
+	const [ isLoading, setIsLoading ] = useState( false );
+	const [ isLoaded, setIsLoaded ] = useState( false );
+	const [ isError, setIsError ] = useState( false );
+	const [ remoteConfig, setRemoteConfig ] = useState( {} );
+
+	const fetchData = async () => {
+		setIsLoading( true );
+
+		try {
+			const result = await getRemoteConfig().finally( () => {
+				setIsLoaded( true );
+				setIsLoading( false );
+			} );
+
+			if ( ! result.config ) {
+				throw new Error( 'Invalid remote config' );
+			}
+
+			setRemoteConfig( result.config );
+		} catch ( error ) {
+			setIsError( true );
+			setIsLoaded( true );
+			setIsLoading( false );
+		}
+	};
+
+	if ( ! isLoaded && ! isLoading ) {
+		fetchData();
+	}
```

Comment:
useEffect on mount + remove the `isLoaded`?

Reply:
It's like here:
https://github.com/elementor/elementor/blob/6e2dd8cf3998803e7304aeb005899ca6cd26af32/modules/ai/assets/js/editor/hooks/use-user-info.js#L21-L35